packages/client/src/containers/LandingPage/CheckinQuestionsPage/AddCheckinQuestionModal.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const AddCheckinQuestionModal = ({
+  showAddModal,
+  setShowAddModal,
+  newQuestionText,
+  setNewQuestionText,
+  handleAdd,
+}) => {
+  if (!showAddModal) {
+    return null;
+  }
+
+  return (
+    <div>
+      <h2>Add Checkin Question</h2>
+      <textarea
```

Comment:
Please change the elements to `material-ui` elements. You can ask @rumana-aafrin about them or see her code. She implemented the first changes with the library.

Reply:
There should be components for the `textarea`, `Typography` and `Button`